lib/killbill/gen/api/pagination.rb

```diff
@@ -41,7 +41,7 @@ def initialize()
 
         def to_java()
           # conversion for iterator [type = java.util.Iterator]
-          @iterator = Killbill::Plugin::Model::EnumeratorIterator.new(@iterator)
```

Comment:
Hi @pierre , any updates? Please inform me if this change is not necessary, I will update my pr. Thanks.